definitions/variable/technology/electricity-operation.yaml

```
@@ -12,360 +12,82 @@
-    unit: MWh
-- Operation Cost|Electricity|Biomass:
-    description: Operation Cost of Biomass power plants
-    unit: EUR_2020
-    skip-region-aggregation: true
-- Operation Cost|Electricity|Biomass|Traditionnal:
-    description: Operation Cost of  Traditionnal Biomass power plants biomass from
-      local and pre-existing resources mostly waste :domestic and agricultural waste,
-      forest residues.
-    unit: EUR_2020
-    skip-region-aggregation: true
-- Operation Cost|Electricity|Biomass|New and imported:
-    description: Operation Cost of New+Imported biomass power plants biomass from
-      dedicated energy crops and biomass imports.
-    unit: EUR_2020
-    skip-region-aggregation: true
-- Operation Cost|Electricity|Nuclear:
-    description: Operation Cost of nuclear power plants
-    unit: EUR_2020
-    skip-region-aggregation: true
-- Operation Cost|Electricity|Geothermal:
-    description: Operation Cost of Geothermal power plants
-    unit: EUR_2020
-    skip-region-aggregation: true
-- Operation Cost|Electricity|Gas:
-    description: Operation Cost of Gas power plants
-    unit: EUR_2020
-    skip-region-aggregation: true
-- Operation Cost|Electricity|Gas|OCGT:
-    description: Operation Cost of Open Cycle Gas Turbine power plants
-    unit: EUR_2020
-    skip-region-aggregation: true
-- Operation Cost|Electricity|Gas|CCGT:
-    description: Operation Cost of Combined Cycle Gas Turbine power plants
-    unit: EUR_2020
-    skip-region-aggregation: true
-- Operation Cost|Electricity|Coal:
-    description: Operation Cost of coal power plants
-    unit: EUR_2020
-    skip-region-aggregation: true
-- Operation Cost|Electricity|Lignite:
-    description: Operation Cost of Lignite power plants
-    unit: EUR_2020
-    skip-region-aggregation: true
-- Operation Cost|Electricity|Oil:
-    description: Operation Cost of Oil power plants
-    unit: EUR_2020
-    skip-region-aggregation: true
-- Operation Cost|Electricity|Hydro|Reservoir:
-    description: Operation Cost of "simple Hydro reservoir " units
-    unit: EUR_2020
-    skip-region-aggregation: true
-- Operation Cost|Electricity|Hydro|Pumped Storage:
-    description: Operation Cost of "Pumped Storage " units
-    unit: EUR_2020
-    skip-region-aggregation: true
-- Operation Cost|Electricity|Hydro|Run of River:
-    description: Operation Cost of "Run of River" units
-    unit: EUR_2020
-    skip-region-aggregation: true
-- Operation Cost|Electricity|Hydro|Ocean:
-    description: Operation Cost of Ocean Power plants
-    unit: EUR_2020
-    skip-region-aggregation: true
-- Operation Cost|Electricity|Solar:
-    description: Operation Cost of Photovoltaic power plants
-    unit: EUR_2020
-    skip-region-aggregation: true
-- Operation Cost|Electricity|Wind|OnShore:
-    description: Operation Cost of  Wind Onshore power plants
+    unit:
+    - MWh
+    - GWh
+
+- Operation Cost|Electricity|<Fuel>:
+    description: Operation Cost of <this fuel> power plants
```

The tags now work differently because we switched to the stand-alone **nomenclature** package.
```suggestion
    description: Operation Cost of <Fuel> power plants
```